threshold_tools_event_types.ipynb

```diff
@@ -0,0 +1,505 @@
+    "\n",
+    "To learn more about the data available on the Analytics Engine, see our [data catalog](https://analytics.cal-adapt.org/data/). The *getting_started.ipynb* notebook contains additional explanations of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.select()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Retrieve data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run `app.retrieve()` to load the data selected above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hourly_data = app.retrieve()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the following analysis, we will work with a subset of the data from just one model simulation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hourly_data_subset = hourly_data.isel(simulation=0, scenario=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To speed up the following computations, load the selected subset of data into memory. This step may take a few minutes to compute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hourly_data_subset = app.load(hourly_data_subset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='transform'></a>\n",
+    "## Step 2: Get block maxima series for different event types\n",
+    "\n",
+    "To facilitate custom definitions of different types of extreme events, there are optional keyword arguments to the `get_block_maxima` function to pull the maximum annual value. The keyword options are:\n",
+    "- `duration`: how long an event continuously lasts in hours (see example 2)\n",
+    "- `groupby` and  `grouped_duration`: work together to define multi-day events (see examples 3 and 4)\n",
+    "\n",
+    "The following four examples show how to use these options to construct different types of events of interest. See section 3 for further analysis and discussion of these events."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 1: hottest hour in each year\n",
+    "The basic use case of `get_block_maxima` pulls the maximum value in each year of data, shown here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams = get_block_maxima(hourly_data_subset)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams.plot()"
```

Comment:
This plot needs more explanation - ideally write in markdown that its plotting the hottest hour per year

Reply:
Also, for me the scenario got cut off in the title

Reply:
will fix this!